src/routes/oauth/utils.ts

```diff
@@ -138,8 +138,7 @@ export const createGrantConfig = (): GrantConfig =>
     },
     {
       defaults: {
-        origin: ENV.AUTH_SERVER_URL,
-        prefix: OAUTH_ROUTE,
+        prefix: `${ENV.AUTH_API_PREFIX}${OAUTH_ROUTE}`,
```

Comment:
How come we need both the the `AUTH_API_PREFIX` and `OAUTH_ROUTE` here?

Reply:
You need to pass to grant the entire prefix where routes will be mounted. If we pass  `OAUTH_ROUTE` we are only passing part of the route.